### PR TITLE
Updates webapp to latest versions of React, React-Router and Babel

### PR DESCRIPTION
**Pull request by btburton42**
• removes outdates files from git
  • cleans up syntax violations and updates linting

### CODE REVIEW

**Comment by btburton42** [comment]
test slack integration

**Comment by btburton42** [comment]
*\* also sets up ground work for moving to redux router and immutable js though this is far from laid in.

File: test/test_helper.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import chai from 'chai';
```
Comment (mngyuan) [inline test/test_helper.js:1]: not familiar at all with chai, but i guess i can cross that bridge when i come to it

**Comment by mngyuan** [comment]
damn, didn't mean to let this sit over the weekend. sorry if that blocked you - looked over it real quick when it was pushed and it just looked like updates because of changing react-router / eslint / babel / etc. API changes. just looked again and it all looks in good order to me. :+1: 

so - let's merge this?

**Comment by btburton42** [comment]
no sweat man. i wasn't blocked at all. hope you had a good weekend. i'll go ahead and merge.

File: test/test_helper.js (hunk 1 above)
Comment (btburton42) [inline test/test_helper.js:1]: chai api is pretty straightforward. this is the tutorial i'm working off of to update the way we're using react components and eventually to setup up redux / immutablejs.
http://teropa.info/blog/2015/09/10/full-stack-redux-tutorial.html

**Pull request merged**